src/main/java/org/zalando/planb/provider/ScopeService.java

```diff
@@ -0,0 +1,62 @@
+package org.zalando.planb.provider;
+
+import com.google.common.collect.ImmutableSet;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.zalando.planb.provider.realms.ClientRealm;
+
+import java.util.Collections;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.joining;
+
+@Component
+public class ScopeService {
+
+    private static final String SPACE = " ";
+
+    @Autowired
+    private ScopeProperties scopeProperties;
+
+    public Set<String> getDefaultScopesForClient(final ClientRealm clientRealm, final String clientId) {
+        return clientRealm
+                .get(clientId)
```

Comment:
This triggers an extra Cassandra call, I think we should only do one call to load client data..